plugins/agendas/README.md

```diff
@@ -0,0 +1,136 @@
+# Agendas Plugin
+
+Generate structured meeting agendas to streamline team collaboration and decision-making processes.
+
+## Features
+
+- 📋 **Outcome Refinement Agendas** - Analyze OCPSTRAT outcome issues and generate structured refinement meeting agendas
+- 🚨 **Issue Hygiene Detection** - Automatically identify missing assignments, incorrect issue types, and other routine problems
+- 📊 **Team Overload Analysis** - Detect component teams that may be overloaded based on issue assignments
+- ⏰ **Age-Based Prioritization** - Flag outcomes that have been open too long or stuck in "New" status
+- ✅ **Actionable Output** - Ready-to-use Markdown agendas that can be copied directly into Confluence
+
+## Prerequisites
+
+- Claude Code installed
+- Jira MCP server configured (same as Jira plugin)
+
+### Setting up Jira MCP Server
+
+```bash
+# Add the Atlassian MCP server
+claude mcp add atlassian npx @modelcontextprotocol/server-atlassian
+```
+
+OR you can use an already running Jira MCP Server:
+
+```bash
+# Add the Atlassian MCP server
+claude mcp add --transport sse atlassian http https://localhost:8080/sse
+```
+
+Configure your Jira credentials according to the [Atlassian MCP documentation](https://github.com/modelcontextprotocol/servers/tree/main/src/atlassian).
+
+### Running Jira MCP Server locally with podman
+
+```bash
+# Start the atlassian mcp server using podman
+podman run -i --rm -p 8080:8080 -e "JIRA_URL=https://issues.redhat.com" -e "JIRA_USERNAME" -e "JIRA_API_TOKEN" -e "JIRA_PERSONAL_TOKEN" -e "JIRA_SSL_VERIFY" ghcr.io/sooperset/mcp-atlassian:latest --transport sse --port 8080 -vv
+```
+
+#### Getting Tokens
+You'll need to generate your own tokens:
+
+- For JIRA API TOKEN, use https://id.atlassian.com/manage-profile/security/api-tokens
+- For JIRA PERSONAL TOKEN, use https://issues.redhat.com/secure/ViewProfile.jspa?selectedTab=com.atlassian.pats.pats-plugin:jira-user-personal-access-tokens
+
+## Installation
+
+### From the OpenShift AI Helpers Marketplace
+
+```bash
+# Add the marketplace (one-time setup)
+/plugin marketplace add https://raw.githubusercontent.com/openshift-eng/ai-helpers/main/marketplace.json
+
+# Install the plugin
+/plugin install agendas
+```
+
+### Manual Installation
+
+```bash
+# Clone the repository
+git clone https://github.com/openshift-eng/ai-helpers.git
+
+# Copy to Claude Code plugins directory
+cp -r ai-helpers/plugins/agendas ~/.claude/plugins/
+
+# Enable the plugin
+/plugin enable agendas
+```
+
+## Available Commands
+
+### `/agendas:outcome-refinement` - Outcome Refinement Meeting Agenda
+
+Analyze OCPSTRAT outcome issues and generate a structured meeting agenda for outcome refinement sessions. The command automatically identifies common issues that require human follow-up and organizes them into actionable discussion points.
+
+**Usage:**
+```bash
+/agendas:outcome-refinement
```

Comment:
how about we include this in a [## Examples](https://github.com/openshift-eng/ai-helpers/blob/main/plugins/hello-world/commands/echo.md#examples) section within the command spec?

Reply:
Added.